source/macos/AdaptiveCards/AdaptiveCards/Views/ACRContentStackView.swift

```diff
@@ -161,12 +163,16 @@ class ACRContentStackView: NSView, ACRContentHoldingViewProtocol, SelectActionHa
         }
     }
     
-    func addSeperator(_ separator: Bool) {
+    func addSeperator(_ separator: Bool, withSpacing space: ACSSpacing) {
         guard separator else { return }
+        var spaceAdded = CGFloat(truncating: HostConfigUtils.getSpacing(space, with: hostConfig))
         let seperatorConfig = hostConfig.getSeparator()
         let lineThickness = seperatorConfig?.lineThickness
         let lineColor = seperatorConfig?.lineColor
+        spaceAdded -= CGFloat(truncating: lineThickness ?? .init(value: 0))
+        addSpacing(spacing: spaceAdded / 2)
         addSeperator(thickness: lineThickness ?? 1, color: lineColor ?? "#EEEEEE")
+        addSpacing(spacing: spaceAdded / 2)
```

Comment:
Change how this function is written, if the first bool always has to be true for function to do anything, then no use of that argument

Reply:
Yes, sure.

Reply:
✅